business_requirement_deliverable_project/wizard/generate_projects.py

```diff
@@ -166,9 +166,9 @@ def _prepare_project_vals(self, br, parent):
             description = br.description
             privacy_visibility = br.project_id.privacy_visibility \
                 or br.project_id._defaults['privacy_visibility']
-            br_id = br.id
-        elif br._name == 'business.requirement.deliverable':
-            br_id = br.business_requirement_id.id
+        #     br_id = br.id
```

Comment:
@victormartinelicocorp 
Your PR looks strange: you comment removed lines.
I would say: remove dead code, do not comment.
Something I missed?